Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -0,0 +1,77 @@
+    radius: 3.5
+    energy: 0.5
+  - type: RadarBlip
+    scale: 0.5
+    requireNoGrid: true
+
+- type: entity
+  id: 50CartridgeBase
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Tag
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)